be/api/po-app/src/main/resources/application.yaml

```diff
@@ -34,6 +34,8 @@ api:
   service:
     username: ${API_SERVICE_USERNAME:user}
     password: ${API_SERVICE_PW:password}
+  scheduler:
+    cron: "*/10 * * * * *"
```

Comment:
Nastaveno na 10s, vime jaky profil bude v produkci? prod

Chceme jeden yaml soubor nebo vic? (oboji jde)

Reply:
zatím bych nechal 1 yaml, ale udělal bych to konfigurovatelne pomoci ENV parametru s tim, ze ten actualni cron bude jako default